packages/nextra-theme-docs/src/styles.css

```diff
@@ -500,6 +500,38 @@ article {
       @apply rotate-90;
     }
   }
+
+  /* Typesettings */
+  &.nextra-body-typesetting-article {
```

Comment:
I guess it's me who removed it because I didn't found class usage due interpolation

Reply:
made rollback in my PR with following change
```ts
      className={cn(
        'nextra-body relative flex w-full min-w-0 max-w-full justify-center pb-8 pr-[calc(env(safe-area-inset-right)-1.5rem)]',
        {
          default: '',
          article: 'nextra-body-typesetting-article'
        }[themeContext.typesetting]
      )}
```

Reply:
Yeah I added that "typesetting" option just for long articles, as they kinda need a little different design than docs.